internal/config/config.go

```diff
@@ -52,10 +52,12 @@ type config struct {
 	GitlabClientKey string `yaml:"gitlab.client-key" env:"OG_GITLAB_CLIENT_KEY"`
 	GitlabSecret    string `yaml:"gitlab.secret" env:"OG_GITLAB_SECRET"`
 	GitlabUrl       string `yaml:"gitlab.url" env:"OG_GITLAB_URL"`
+	GitlabName      string `yaml:"gitlab.service-name" env:"OG_GITLAB_SERVICE_NAME"`
 
 	GiteaClientKey string `yaml:"gitea.client-key" env:"OG_GITEA_CLIENT_KEY"`
 	GiteaSecret    string `yaml:"gitea.secret" env:"OG_GITEA_SECRET"`
 	GiteaUrl       string `yaml:"gitea.url" env:"OG_GITEA_URL"`
+	GiteaName      string `yaml:"gitea.service-name" env:"OG_GITEA_SERVICE_NAME"`
```

Comment:
For clarity perhaps it might be better to have these fields named GiteaServiceName to match the yaml field?

Reply:
I apologise, that was unclear. My exact suggestion is this:
```diff
diff --git a/docs/configuration/cheat-sheet.md b/docs/configuration/cheat-sheet.md
index 756cabc..46f7f6c 100644
--- a/docs/configuration/cheat-sheet.md
+++ b/docs/configuration/cheat-sheet.md
@@ -24,11 +24,11 @@
 | gitlab.client-key     | OG_GITLAB_CLIENT_KEY     | none                  | The client key for the GitLab OAuth application.
                                                               |
 | gitlab.secret         | OG_GITLAB_SECRET         | none                  | The secret for the GitLab OAuth application.
                                                               |
 | gitlab.url            | OG_GITLAB_URL            | `https://gitlab.com/` | The URL of the GitLab instance.

                                                               |
-| gitlab.service-name   | OG_GITLAB_SERVICE_NAME   | `GitLab`              | The name of the GitLab instance. It is displayed in the OAuth login button.
                                                                                                           |
+| gitlab.name           | OG_GITLAB_NAME           | `GitLab`              | The name of the GitLab instance. It is displayed in the OAuth login button.
                                                                                                           |
 | gitea.client-key      | OG_GITEA_CLIENT_KEY      | none                  | The client key for the Gitea OAuth application.
                                                               |
 | gitea.secret          | OG_GITEA_SECRET          | none                  | The secret for the Gitea OAuth application.
                                                               |
 | gitea.url             | OG_GITEA_URL             | `https://gitea.com/`  | The URL of the Gitea instance.

                                                               |
-| gitea.service-name    | OG_GITEA_SERVICE_NAME    | `Gitea`               | The name of the Gitea instance. It is displayed in the OAuth login button.
                                                                                                          |
+| gitea.name            | OG_GITEA_NAME            | `Gitea`               | The name of the Gitea instance. It is displayed in the OAuth login button.
                                                                                                          |
 | oidc.client-key       | OG_OIDC_CLIENT_KEY       | none                  | The client key for the OpenID application.
                                                               |
 | oidc.secret           | OG_OIDC_SECRET           | none                  | The secret for the OpenID application.    
                                                               |
 | oidc.discovery-url    | OG_OIDC_DISCOVERY_URL    | none                  | Discovery endpoint of the OpenID provider.
                                                               |
diff --git a/internal/config/config.go b/internal/config/config.go
index 3e77f71..5f3a65e 100644
--- a/internal/config/config.go
+++ b/internal/config/config.go
@@ -52,12 +52,12 @@ type config struct {
        GitlabClientKey string `yaml:"gitlab.client-key" env:"OG_GITLAB_CLIENT_KEY"`
        GitlabSecret    string `yaml:"gitlab.secret" env:"OG_GITLAB_SECRET"`
        GitlabUrl       string `yaml:"gitlab.url" env:"OG_GITLAB_URL"`
-       GitlabName      string `yaml:"gitlab.service-name" env:"OG_GITLAB_SERVICE_NAME"`
+       GitlabName      string `yaml:"gitlab.name" env:"OG_GITLAB_SERVICE_NAME"`

        GiteaClientKey string `yaml:"gitea.client-key" env:"OG_GITEA_CLIENT_KEY"`
        GiteaSecret    string `yaml:"gitea.secret" env:"OG_GITEA_SECRET"`
        GiteaUrl       string `yaml:"gitea.url" env:"OG_GITEA_URL"`
-       GiteaName      string `yaml:"gitea.service-name" env:"OG_GITEA_SERVICE_NAME"`
+       GiteaName      string `yaml:"gitea.name" env:"OG_GITEA_SERVICE_NAME"`

        OIDCClientKey    string `yaml:"oidc.client-key" env:"OG_OIDC_CLIENT_KEY"`
        OIDCSecret       string `yaml:"oidc.secret" env:"OG_OIDC_SECRET"`
```